index.html

```diff
@@ -23,28 +23,36 @@
       function gtag() {
         dataLayer.push(arguments);
       }
-      gtag("js", new Date());
+      gtag('js', new Date());
 
-      gtag("config", "UA-214517820-1");
+      gtag('config', 'UA-214517820-1');
     </script>
 
     <title>Akins Portfolio</title>
   </head>
   <body>
     <header class="header">
-      <section class="logo-section">
+      <div class="logo-section">
         <a href="#" class="my-logo">My Logo</a>
         <nav class="nav">
-          <span class="iconify" data-icon="eva:menu-outline"></span>
           <ul class="nav-ul">
             <li><a href="#">Portfolio </a></li>
-            <li><a href="#">About </a></li>
-            <li><a href="#">Contact </a></li>
+            <li><a href="#about">About </a></li>
+            <li><a href="#contact">Contact </a></li>
           </ul>
         </nav>
-      </section>
+        <div class="menu"><span></span><span></span><span></span></div>
+        <!-- Mobile menu Section -->
+        <nav class="mobile-menu">
+          <ul>
```

Comment:
- [ ] The mobile menu looks awesome, however I noticed that the `x` button is missing. When the user clicks (or taps) the close (X) button, the mobile menu should disappear. Please add the `x` button 

 Figma |  Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/146580205-522f573d-603c-4dd5-9c9a-385e8ce3d2f7.png)  | ![image](https://user-images.githubusercontent.com/77038610/146580278-183bcd07-32dd-40d1-bc9c-aec4348e44b1.png)

- [ ] [OPTIONAL] I have noticed that when users click the mobile menu, they can still scroll the website, which doesn't look good, it is very easy to fix, on hamburger click you can add 
`body{ overflow: hidden }`, 
and remove it when the modal is closed. in this case users will not be able to scroll: 

https://user-images.githubusercontent.com/77038610/146581223-97849606-9915-4b2a-8ef6-925d36041dd8.mp4